### ISSUE TITLE
Failed building

### ISSUE THREAD

**Issue by y0shir** [axle-h/baby-smash#2]
I am running debian testing and i got a following error while going through compilation steps:

```
[ 72%] Building CXX object CMakeFiles/baby-smash.dir/src/display/Display.cpp.o
/home/USERNAME/builds/baby-smash/src/assets/Assets.cpp: In function ‘std::string getFolder(AudioAssetType)’:
/home/USERNAME/builds/baby-smash/src/assets/Assets.cpp:94:1: warning: control reaches end of non-void function [-Wreturn-type]
   94 | }
      | ^
In file included from /home/USERNAME/builds/baby-smash/src/display/Display.h:5,
                 from /home/USERNAME/builds/baby-smash/src/display/Display.cpp:1:
/home/USERNAME/builds/baby-smash/src/display/../media/FrameService.h:12:3: error: ‘AVCodecContext’ does not name a type; did you mean ‘AVIODirContext’?
   12 |   AVCodecContext *dec_ctx;
      |   ^~~~~~~~~~~~~~
      |   AVIODirContext
gmake[3]: *** [CMakeFiles/baby-smash.dir/build.make:118: CMakeFiles/baby-smash.dir/src/display/Display.cpp.o] Ошибка 1
gmake[3]: *** Ожидание завершения заданий…
gmake[2]: *** [CMakeFiles/Makefile2:260: CMakeFiles/baby-smash.dir/all] Ошибка 2
gmake[1]: *** [CMakeFiles/Makefile2:267: CMakeFiles/baby-smash.dir/rule] Ошибка 2
gmake: *** [Makefile:190: baby-smash] Ошибка 2
```

**Comment by axle-h** [comment]
sorry missed this, thanks for reporting! looks to be ffmpeg 5.x. I've been using Ubuntu LTS so not hit this yet. The changes look breaking... I didn't even spot this was deprecated so maybe the entire package was deprecated, I'll take a look.

**Comment by axle-h** [comment]
OK, I just merged something that may help #3 
If you're still interested, let me know if this worked!
